src/jmbitcoin/secp256k1_transaction.py

```diff
@@ -184,6 +184,15 @@ def tx_vsize(tx):
     non_witness_size = raw_tx_size - witness_size
     return ceil(non_witness_size + .25 * witness_size)
 
+def tx_total_outputs_value(tx: CTransaction) -> int:
```

Comment:
Probably isn't needed; you could just write `sum([x.nValue for x in tx.vout])`

Reply:
ca972cd72d8998c24bcc77682c379a1fdd70c0c5